cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -59,8 +59,181 @@ def test_dropdownlist_required_dialog_element(appliance, catalog_item):
     Polarion:
         assignee: nansari
         initialEstimate: 1/4h
+        testtype: functional
         casecomponent: Services
+        startsin: 5.10
+        tags: service
     """
     service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
     view = navigate_to(service_catalogs, 'Order')
     assert view.submit_button.disabled
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_text_box_required_dialog_element():
```

Comment:
Question: When these are automated will they be parametrized? I see that most of these have the same testSteps.

Reply:
It can't be parametrized.If possible I will parametrized these test while writing automation.